cmd/notary/config.json

```diff
@@ -0,0 +1,7 @@
+{
+	"remote_server": {
+		"url": "https://localhost:4443",
```

Comment:
the problem is that notary-server isn't set in /etc/hosts by default, and I wanted something that works out of the box.